fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,42 +1,42 @@
-WordPress for Android puts the power of web publishing in your pocket. It’s a website creator and so much more!
+<b>WordPress for Android</b> puts the power of web publishing in your pocket. It’s a website creator and so much more!
```

Comment:
ℹ️  Notice how:
 - The [copy which is in `WordPress/metadata/full_description.txt`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/metadata/full_description.txt) — which [this file is copied from](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/fastlane/Fastfile#L724-L727) — does NOT have HTML tags
 - While the copy used in the PlayStore (which was pulled here via f25c012b7d1d8870ea6894e358019a966ac7e3a3 and is thus what you see here in this diff) DOES have tags.

So at some point those HTML tags were stripped, and the stripped copy got embedded in `PlayStoreStrings.po` and made its way to GlotPress, which led to the translations (which later got uploaded to PlayStore) also missing the HTML tag. So in the end, currently in the PlayStore, only the English copy has HTML tags, but other locales don't… 🤷 

That's not really a big deal, since the short and full description copies will be updated soon anyway [ref: pbvYe0-JV-p2], so decided to not spend too much time on this; but just wanted to explain the diff we're seeing here.